packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain/SourceNetworkBox.tsx

```diff
@@ -178,6 +178,22 @@ export function SourceNetworkBox({
           value={isMaxAmount ? '' : amount}
         />
 
+        {process.env.NEXT_PUBLIC_FEATURE_BATCH_ERC20_ETH === 'true' &&
+          selectedToken && (
```

Comment:
need to check if this should be allowed at all for teleporter

Reply:
Disabled teleporter for now and will look into it more